src/main/scripts/mlsearch

```diff
@@ -6,67 +6,101 @@ import os
+ap.add_argument("-ck", "--cookies", required=True, help="Cookies of current user.")
+ap.add_argument(
+    "-tm", "--timestamp", required=True, help="Timestamp of requesting API."
+)
+ap.add_argument(
+    "-pu", "--pwc_user", required=False, help="Paper with code repository user name."
+)
+ap.add_argument(
+    "-pp", "--pwc_password", required=False, help="Paper with code repository password."
+)
+ap.add_argument(
+    "-gt", "--github_acc_token", required=False, help="Github access token."
+)
+ap.add_argument(
+    "-yk", "--youtube_dev_key", required=False, help="Youtube developer key."
+)
+ap.add_argument(
+    "-yntp",
+    "--y_next_page_token",
+    required=False,
+    help="Next page token for Youtube API.",
+)
+ap.add_argument(
+    "-yo", "--y_query_order", required=False, help="Youtube Query Order."
+)
 args = vars(ap.parse_args())
 
+
 def main(event):
     headers = {
-            'Access-Control-Allow-Origin': '*',
-            'X-Requested-With': '*',
-            'Access-Control-Allow-Headers': 'Content-Type,X-Amz-Date,Authorization,X-Api-Key,x-requested-with',
-            'Access-Control-Allow-Methods': 'OPTIONS,POST,GET'
-        }
+        "Access-Control-Allow-Origin": "*",
+        "X-Requested-With": "*",
+        "Access-Control-Allow-Headers": "Content-Type,X-Amz-Date,Authorization,X-Api-Key,x-requested-with",
+        "Access-Control-Allow-Methods": "OPTIONS,POST,GET",
+    }
     try:
         param_names = [
-            'query', 'init_idx',
-            'count', 'source',
-            'cookies', 'timestamp',
-            'y_next_page_token']
-        response_msg = hp.response('success', 200)
+            "query",
+            "init_idx",
+            "count",
+            "source",
+            "cookies",
+            "timestamp",
+            "y_next_page_token",
+        ]
+        response_msg = hp.response("success", 200)
         if hp.is_valid_parameters(event, param_names):
             params = hp.parse_parameters(event)
             if params.values():
                 api_request = APIRequest(
-                    params['source'], 
-                    params['query'], 
-                    params['init_idx'], 
-                    params['count'],
-                    params['y_next_page_token'])
-                if 'pwc_user'in event and 'pwc_password' in event:
-                    api_request.pwc_auth_info = (event['pwc_user'], event['pwc_password'])
-                if 'github_acc_token' in event:
-                    api_request.github_acc_token = event['github_acc_token']
-                if 'youtube_developer_key' in event:
-                    api_request.youtube_developer_key = event['youtube_developer_key']
+                    params["source"],
+                    params["query"],
+                    params["init_idx"],
+                    params["count"],
+                    params["y_next_page_token"],
```

Comment:
First, I was thinking the `,` is not really necessary. But it just a matter of style so, it's okay.